Robert can't count

kerberosmansour opened the pull request:
changed "If 30 votes are cast, a two thirds vote is 30" to If 30 votes are cast, a two thirds vote is 20"

kingthorin commented:
Thanks for this, however, I wonder if there's an even better way this could be handled. With collections or something. Having to create a boiler plate `.md` to accompany each recording isn't great.

Or maybe a naming convention needs to be established for the recordings and a github action setup to create the related markdown file?

mikemccamon commented:
I'm not a big fan of cluttering up the sidecol, but I very glad someone took the time to add recordings to historical meeting tab. @hblankenship  I'm not an expert but I cool with commiting some of this code.

mikemccamon commented:
@kingthorin this issue was closed. We don't need the md files. We have a script that loops through a folder listing file names.

kingthorin commented:
Thanks, you'd re-opened this 18 days ago, which was 9 days following my previous comment which you appear to be replying to. I'll leave it to you to actually close....

kerberosmansour committed:
Robert can't count

changed "If 30 votes are cast, a two thirds vote is 30" to If 30 votes are cast, a two thirds vote is 20"
